ci-operator/config/openshift/release/openshift-release-master__ci-4.7-upgrade-from-stable-4.6.yaml

```
@@ -90,13 +90,13 @@ tests:
 - as: e2e-openstack-upgrade
   interval: 48h
   steps:
-    cluster_profile: openstack
+    cluster_profile: openstack-vh-mecha 
```

```suggestion
    cluster_profile: openstack-vh-mecha
```

Extra space breaks generated job verification

---

Thanks!